dev/user.clj

```diff
@@ -117,6 +118,41 @@
        sql/parse
        (query-db db-name)))
 
+(defn load-chat-ledger
+  []
+  (let [collection-txn  [{:_id  "_collection"
+                          :name "person"}
+                         {:_id  "_collection"
+                          :name "chat"}
+                         {:_id  "_collection"
+                          :name "comment"}
+                         {:_id  "_collection"
+                          :name "artist"}
+                         {:_id  "_collection"
+                          :name "movie"}]
+
+        pred-filename   "../test/fluree/db/ledger/Resources/ChatApp/chatPreds.edn"
+        predicate-txn   (edn/read-string (slurp (io/resource pred-filename)))
+
+        data-filename   "../test/fluree/db/ledger/Resources/ChatApp/chatAppData.edn"
+        data-txn        (edn/read-string (slurp (io/resource data-filename)))]
+    (log/info "Creating new ledger")
+    @(fdb/new-ledger (:conn system) test-helpers/ledger-chat)
+    (Thread/sleep 250)
+
+    (log/info "Adding collections")
+    (async/<!! (fdb/transact-async (:conn system) test-helpers/ledger-chat collection-txn))
```

Comment:
Unless you are working in a go-loop, I prefer the non-asyc API:
`@(fdb/transact (:conn system) test-helpers/ledger-chat collection-txn)`